seed/analysis_pipelines/better.py

```diff
@@ -445,38 +448,21 @@ def _better_building_service_request(bsync_xml):
     :param bsync_xml: BSync xml for property
     :returns: requests.Response building_id
     """
-    url = "https://better-lbnl-development.herokuapp.com/api/v1/buildings/"
-
-    # TODO: Make this the BSync file once my PR is merged
-    bsync_xml = {
-        "name": "SEED test",
-        "space_type": "OFFICE",
-        "floor_area": 10000,
-        "location": "Denver",
-        "utility_bills": [
-            {
-                "fuel_type": "ELECTRIC_GRID",
-                "bill_start_date": "2019-06-07",
-                "bill_end_date": "2019-06-07",
-                "consumption": 100,
-                "unit": "KWH",
-                "cost": 10,
-                "temperature": 50
-            }
-        ]
-    }
+    url = "{host}/api/v1/buildings/".format(host=HOST)
+
+    with open(bsync_xml, 'r') as file:
+        bsync_content = file.read()
 
     headers = {
-        'accept': 'application/json',
         'Authorization': settings.BETTER_TOKEN,
-        'Content-Type': 'application/json',
+        'Content-Type': 'buildingsync/xml',
     }
     try:
-        response = requests.request("POST", url, headers=headers, data=json.dumps(bsync_xml))
+        response = requests.request("POST", url, headers=headers, data=bsync_content)
```

Comment:
Documenting our discussion, it might be nice to check `response.status_code` is 2xx before trying to parse the response